src/test/resources/features/login.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Login with a valid create parameters
+  Scenario: Login
+    Given I visit the "login" page of Pivotal Tracker
+    When I fill the field "username or email"
```

Comment:
It's better is you put the  user from where we gets username and password because the credentials will be readed from a json file with the credentials